### PR TITLE
[ChainOps] Updates in the block-explorer configmap

### PR DESCRIPTION
**Pull request by ilchebedelovski**

### CODE REVIEW

File: deploy/helm/block-explorer/templates/configmap-settings.yaml [hunk 1]
```diff
@@ -21,45 +21,44 @@ data:
...
                     "displayNamePlural": "CHOTS",
                     "fraction": 1000000
                 }
-
             ],
             "gasPrice": 0.02,
             "coingeckoId": "sif"
         },
-        "genesisFile": "https://raw.githubusercontent.com/Sifchain/networks/feature/genesis/testnet/monkey-bars-testnet-3/genesis.json",
+        "genesisFile": "https://raw.githubusercontent.com/Sifchain/networks/feature/genesis/testnet/latest.json",
```
Comment (intl-man) [inline deploy/helm/block-explorer/templates/configmap-settings.yaml:61]: For now this works. At some state we will need to parameterise this and the other options.

**Review by intl-man — APPROVED** [review]

File: deploy/helm/block-explorer/templates/configmap-settings.yaml [hunk 2]
```diff
@@ -21,45 +21,44 @@ data:
...
             "powerReduction": 1000000,
             "coins": [
                 {
-                    "denom": "trwn",
-                    "displayName": "TRWN",
-                    "displayNamePlural": "TRWNS",
+                    "denom": "rowan",
+                    "displayName": "ROWAN",
+                    "displayNamePlural": "ROWAN",
                     "fraction": 1000000
```
Comment (intl-man) [inline deploy/helm/block-explorer/templates/configmap-settings.yaml:31]: I think these factions are wrong. These look like 1e6 to me. The precision we use is 1e8. I think these all need to be changed to `100000000`.

**Review by intl-man — CHANGES_REQUESTED** [review]

File: deploy/helm/block-explorer/templates/configmap-settings.yaml (hunk 2 above)
Comment (utx0) [reply deploy/helm/block-explorer/templates/configmap-settings.yaml:31]: Hey actually its 1e18 due to the way peggy tokens are getting minted from eth:

1 eth = 1000000000000000000 wei
1 eth = 1e18 wei

Comment (utx0) [reply deploy/helm/block-explorer/templates/configmap-settings.yaml:31]: https://eth-converter.com/

**Review by intl-man — APPROVED** [review]

**Pull request merged**